src/app.view.html

```diff
@@ -2,15 +2,15 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title><?= $title ?></title>
+  <title>Donate to Keyman</title>
   <base href="/">
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
-  <script src="https://checkout.stripe.com/checkout.js"></script>
+  <script src="https://checkout.stripe.com/checkout.js" data-key="pk_test_g239FhjT812uAnrOdQzUxOrp"></script>
```

Comment:
You've still got a key here.

Reply:
Thanks Marc. That key has been deactivated as well.